docs/about/astro.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Astro portal
+description: Fundamental white label web components for building your design system
+---
+
+# Astro Portal Announcement
+
+We are excited to introduce the new version of our portal, now built with the Astro framework! You can access it at <a href="https://lion.js.org/next" rel="noopener noreferrer">/next</a>.
+
+## What’s New?
+
+- **Modern Look & Feel:** The UI has been redesigned in close collaboration with our designers. Expect further improvements as the design team continues to enhance the user experience.
+- **Search Functionality:** Quickly find components and documentation with the new search feature.
+- **Upgraded Dependencies:** All major dependencies and approaches have been updated for better performance, security, and maintainability.
+
+## Compatibility
+
+The new Astro portal is fully compatible with the previous Rocket-based portal. Maintaining the same structure and functionality was a key challenge and priority, ensuring a seamless experience for all users. Both portals will run in parallel until around the new year 2026, after which the Astro portal will become the default.
```

Comment:
```suggestion
The new Astro portal is fully compatible with the previous Rocket-based portal. Maintaining the same structure and functionality was a key challenge and priority, ensuring a seamless experience for all users. Both portals will run in parallel until in the near term future. During the course of 2026 the Astro portal will become the default.
```

Nice write-up 👏.

Maybe we can be a bit more conservative on the "fully-Astro" timelines.
I think it would be best/safest to have that flow working E2E (our extended portal as well)... (unless you have this already working in a branch of course? :))

And ideally we would combine it with quite a few deisgn and content upgrades as well.